lib/compat/wordpress-6.7/rest-api.php

```diff
@@ -114,3 +114,52 @@ function gutenberg_override_default_rest_server() {
 	return 'Gutenberg_REST_Server';
 }
 add_filter( 'wp_rest_server_class', 'gutenberg_override_default_rest_server', 1 );
+
+/**
+ * Adds user avatar URLs to the REST API response for WordPress version 6.7 and below.
+ *
+ * This function is used to add avatar URLs for the author of a post in the REST API response.
+ * It checks if the function 'add_user_avatar_urls_in_rest_response_6_7' does not already exist,
+ * and if not, defines the function.
+ *
+ * @param object $response The REST API response object.
+ * @return object The modified REST API response object with added avatar URLs.
+ */
+if ( ! function_exists( 'add_user_avatar_urls_in_rest_response_6_7' ) && gutenberg_is_experiment_enabled( 'gutenberg-block-comment' ) ) {
+	function add_user_avatar_urls_in_rest_response_6_7( $response ) {
+
+		if ( $response->data['author'] ) {
+			$response->data['author_avatar_urls'] = array(
+				'default' => get_avatar_url( $response->data['author'] ),
+				'48'      => add_query_arg( 's', 48, get_avatar_url( $response->data['author'], array( 'size' => 48 ) ) ),
+				'96'      => add_query_arg( 's', 96, get_avatar_url( $response->data['author'], array( 'size' => 96 ) ) ),
+			);
+		}
+
+		return $response;
+	}
+	add_filter( 'rest_prepare_comment', 'add_user_avatar_urls_in_rest_response_6_7', 10, 1 );
```

Comment:
Priority `10` and `1` accepted arguments are the defaults for `add_filter()`, so they can be omitted.

```suggestion
	add_filter( 'rest_prepare_comment', 'add_user_avatar_urls_in_rest_response_6_7' );
```